dom.bs

```diff
@@ -639,10 +639,18 @@ dictionary EventInit {
 <p>An {{Event}} object is simply named an <dfn export id=concept-event>event</dfn>. It allows for
 signaling that something has occurred, e.g., that an image has completed downloading.</p>
 
+<p>An <a>event</a> has an associated <dfn export for=Event>relatedTarget</dfn> (an
+{{EventTarget}} object). Unless stated otherwise it is null.</p>
+
+<p>An <a>event</a> has an associated <dfn export>set the relatedTarget</dfn> algorithm,
+which takes an {{EventTarget}} <var>relatedTarget</var>. Unless specified otherwise it does
+nothing.</p>
```

Comment:
Can't we just manipulate the "relatedTarget" concept you introduced for events already and define the relatedTarget attribute as reflecting that concept?